_projects/2024/100540803/100540803.Rmd

```diff
@@ -0,0 +1,601 @@
+---
+title: "Global Fish Stock Overfishing Status"
+description: "Replication of a chart illustrating fish capture and abundance relative to sustainable levels."
+author: 
+  name: "Linghan Zheng"
+  date: "Dec.24,2024"
+format: html
+editor: visual
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(out.width="100%", fig.align="center")
+```
+
+## Introduction
+
+Overfishing poses a significant threat to marine ecosystems by causing declining fish populations, reduced fish sizes, and even the potential extinction of species. A fish stock becomes overfished when its abundance (population size) falls below levels needed to sustain its **maximum sustainable yield (MSY)**—the largest amount of fish that can be sustainably captured over the long term.
+
+To better understand fish stock health, Kobe plots are used as a dynamic tool to evaluate the relationship between fish stock capture and abundance relative to the MSY.
+
+-   The *x-axis* represents fish stock abundance relative to the level that can produce the MSY.\
+-   The *y-axis* represents fish stock capture relative to the MSY.
+
+The plot divides fish stocks into four quadrants based on their abundance and capture levels:
+
+1.  **Overfished and currently overfishing**: *Low abundance* and *high capture*.\
+2.  **Not overfished but currently overfishing**: *High capture* but *sufficient abundance*.\
+3.  **Overfished but currently not overfishing**: *Low abundance* and *reduced capture*.\
+4.  **Not overfished and currently not overfishing**: *Sufficient abundance* and *low capture*.
+
+This framework, supported by data from the **US National Oceanic and Atmospheric Administration (NOAA)**, provides insights into long-term sustainability and helps guide fisheries management strategies.
+
+## Original graph
+
+![Kobe Plot 1](images/picture1.png){width="100%"} ![Kobe Plot 2](images/picture2.png){width="100%"} ![Kobe Plot3](images/picture3.png){width="100%"} *Note: The original graph can be found on the [World Bank SDG Atlas - Goal 14: Life Below Water](https://datatopics.worldbank.org/sdgatlas/goal-14-life-below-water/?lang=en#c14s1).*

```

Comment:
Please set the images as "external" as e.g. David does [here](https://github.com/IBiDat/dataviz/blob/81084e40371bd8161998806e9c7471e04ad47cfe/_projects/2024/100535712/100535712.Rmd#L26).